### DISCUSSION TITLE
It is part of the `laravel-package-tools` package, which should be pulled in by composer

### DISCUSSION THREAD

**Discussion by Hishman** [spatie/laravel-backup#1355] [Q&A]
It is part of the `laravel-package-tools` package, which should be pulled in by composer

https://github.com/spatie/laravel-backup/blob/39dc00988cd28e7cd06237dae27bf463fa9420fd/composer.json#L32

_Originally posted by @freekmurze in https://github.com/spatie/laravel-backup/discussions/1323#discussioncomment-815000_

**Comment by Hishman** [comment]
@ freekmurze. Please don't give any answer assuming that the Laravel community has as much knowledge of your package as you do. Would be nice to attach some instructions of how beginners like myself can solve the posted issues. Thanks

**Comment by erikn69** [comment]
If you want a better answer you have to do a better question, 
Also you have to enumerate your environment versions(Version of laravel, version of php, version of package, os), make sure you try the latest versions, it is possible that the problem is already solved(V7 is current version, run `composer update`)

>Would be nice to attach some instructions of how beginners like myself can solve the posted issues

nobody else has that problem, i'm a beginner too, i only follow the docs [laravel-backup docs](https://spatie.be/docs/laravel-backup/v7/introduction)
You don't have to install  laravel-package-tools, composer make everything automatically
